e2e/tests-dfx/info.bash

```diff
@@ -66,3 +66,25 @@ teardown() {
   assert_command dfx info candid-ui-url  
   assert_eq "http://127.0.0.1:$(dfx info webserver-port)/?canisterId=$(dfx canister id __Candid_UI)"
 }
+
+@test "security-policy produces valid json5 headers" {
+  dfx_new_frontend
+  install_asset assetscanister
+  touch src/e2e_project_frontend/assets/thing.json
+
+  dfx_start
+  dfx canister create --all
+
+  echo "[
+    {
+      \"match\": \"**/*\",
+      \"headers\": {
+        $(dfx info security-policy)
```

Comment:
Great test!  However, this test would pass if `dfx info security-policy` produced no output to stdout.  Could we have a separate test that digs into its output a little bit?